### DISCUSSION TITLE
Room not working with hubs cloud in Chrome or Edge - works fine in Firefox and on Mozilla Hub

### DISCUSSION THREAD

**Discussion by m-zainkhawaja** [mozilla/hubs#4403] [Hubs Cloud Questions]
I have created a simple test room with basic geometry - all fields are green (textures, polygon count etc). After adding a few polygons, the **room fails to load on Hubs cloud running in Digital Ocean and gets stuck in a loading loop for Chrome or Edge. The same room with no modifications works fine in Firefox. If the exact same room is deployed to the Mozilla Hubs page, it works great across all browsers.** 

**Specifically, I noticed that when the room loades correctly, the following statement is printed in the consosle:**
physics-utils.js:59 traverseMeshesAndAddShapes
console.js:35 heightfield or trimesh found on scene
console.js:35 Successfully connected to the networked scene.

**However, when it fails to load, the above statements are not printed, and the following errors are shown:**
Releasing uncached gltf ..../files/36c9457f-6f91-4ee7-adab-e6e91f238d2c.bin
Failed to load glTF model
Releasing uncached gltf  ..../files/36c9457f-6f91-4ee7-adab-e6e91f238d2c.bin
eleasing uncached gltf /WKT3QPA/objects.gltf
gltf-model-plus.js:42 Releasing uncached gltf #spawn-point
GET.../files/36c9457f-6f91-4ee7-adab-e6e91f238d2c.bin net::ERR_CONNECTION_CLOSED 200

Any help would be greatly appreciated
Thanks and best wishes

**Comment by miik-miik** [comment]
Hello Hello! I have a similiar issue - do you find a solution for this?

**Reply by m-zainkhawaja** [reply]
Sorry, no fix yet. Perhaps @jdm @brianpeiris @vincentfretin could please advise on this matter? Would really appreciate to know whether it's a bug or something we're doing incorrectly.

**Reply by markusTraber** [reply]
Only a workaround: Compressing the scenery as I described here: https://github.com/mozilla/hubs/issues/4404#issuecomment-980237191
